package.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "ocw-course-hugo-theme",
+  "version": "1.0.0",
+  "description": "This is a Hugo (https://gohugo.io) theme for publishing an MIT OpenCourseWare (https://ocw.mit.edu/) course as a static site",
+  "main": "index.js",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "npm run build:webpack",
+    "build:webpack": "cross-env NODE_ENV=production webpack --config assets/webpack/webpack.prod.js",
+    "lint": "eslint src build-scripts",
+    "fmt": "LOG_LEVEL= prettier-eslint --write --no-semi $PWD/'build-scripts/**/*.js' $PWD/'assets/**/*.js' $PWD/'assets/**/*.scss'",
+    "fmt:check": "LOG_LEVEL= prettier-eslint --list-different --no-semi $PWD/'build-scripts/**/*.js' $PWD/'assets/**/*.js' $PWD/'assets/**/*.scss'",
+    "scss_lint": "node ./node_modules/sass-lint/bin/sass-lint.js --verbose --no-exit ./src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mitodl/ocw-course-hugo-theme.git"
+  },
+  "keywords": [
+    "ocw",
+    "hugo",
+    "course",
+    "static",
+    "site"
+  ],
+  "author": "MIT Open Learning",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mitodl/ocw-course-hugo-theme/issues"
+  },
+  "homepage": "https://github.com/mitodl/ocw-course-hugo-theme#readme",
+  "engines": {
+    "node": "^12.13.0",
+    "yarn": "^1.19.1"
+  },
```

Comment:
what do you think about removing these? especially when not working inside Docker I don't think they're particularly helpful